Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,53 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+# Analysis of Algorithms Answers
+## Exercise I
+
+a) Answer: The time complexity of this pseudo code is O(n^3)
+
+b) Answer: O(n^4)
+<!-- scratch area -->
+L1    L2       L3            L4     
+ n * (n-1) * ((n-1)-1) * (((n-1)-1)-1+10) = O(n^4)
+<!-- End scratch area -->
+
```

Comment:
the last for loop is a constant O(1) and therefore makes the entire thing be O(n^3)